guiEditor/src/tools.ts

```diff
@@ -78,4 +78,20 @@ export class Tools {
             grid.addControl(controls[i], tags[i].x, tags[i].y);
         }
     }
+
+    public static isContainer(guiControl: Control) {

```

Comment:
Why not adding a property to the controls to know if they are a container ? This would prevent to maintain this separately

Reply:
You could then simply do control.isContainer

Reply:
Not a bad idea. I'm down to add it. :)

Reply:
well even simpler: `return guiControl instanceof Container`

Reply:
Is it not used on a frequent code path ? cause instanceof is slow :-)

Reply:
It is only for the editor

Reply:
so yes what he said :-)